### PR TITLE
Optimize some redundant code

### PR DESCRIPTION
**Pull request by changshuchao**
Signed-off-by: changshuchao <chang.shuchao1@zte.com.cn>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Optimize some redundant code.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/api/pod/util.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ const (
 )
 
 // AllContainers specifies that all containers be visited
-const AllContainers ContainerType = (InitContainers | Containers | EphemeralContainers)
+const AllContainers = InitContainers | Containers | EphemeralContainers
```
Comment (thockin) [inline pkg/api/pod/util.go:47]: I think this is a net-negative.  It relies on a subtle facet of Go that even I had to convince myself was correct.  Better to stay explicit, IMO.

What motivated this change?

File: pkg/api/v1/pod/util.go [hunk 2]
```diff
@@ -63,7 +63,7 @@ const (
 )
 
 // AllContainers specifies that all containers be visited
-const AllContainers ContainerType = (InitContainers | Containers | EphemeralContainers)
+const AllContainers = InitContainers | Containers | EphemeralContainers
```
Comment (thockin) [inline pkg/api/v1/pod/util.go:66]: same comment as above

**Comment by changshuchao** [comment]
@thockin Maybe I still don’t understand, but thank you for your reply :)

**Pull request closed without merging**